pages/styles/work-at.js

```diff
@@ -1,6 +1,30 @@
-import css from 'styled-jsx/css'
+import css from "styled-jsx/css";
+import { breakpoints } from "./breakpoints.js";
 
 export default css`
-li { list-style: none; width: 50%; padding: 20px; }
-ul { display: flex; flex-wrap: wrap; padding: 20px;}
-`
+  li {
+    list-style: none;
+    width: 100%;
+  }
+  ul {
+    justify-content: space-between;
+    padding: 0;
+    max-width: 1400px;
+    margin: 0 auto;
+    padding: 40px 30px 60px 30px;
+  }
+
+  @media screen and ${breakpoints.small} {
+    ul {
+      display: -ms-flexbox;
```

Comment:
these are not needed. It seem the nextjs build system comes with autoprefixing out of the box